locales/en/messages.json

```diff
@@ -737,7 +737,91 @@
+        "message": "List of flags indicating why arming is currently not allowed. Pass the mouse over the flag or refer to the Wiki ('Arming Sequence & Safety' page) for more info."
+    },
+    "initialSetupArmingDisableFlagsTooltipNO_GYRO": {
+        "message": "A gyro was not detected",
+        "description": "Description of the NO GYRO arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipFAILSAFE": {
+        "message": "Failsafe is active",
+        "description": "Description of the FAILSAFE arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipRX_FAILSAFE": {
+        "message": "No valid receiver signal is detected",
+        "description": "Description of the RX_FAILSAFE arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipBAD_RX_RECOVERY": {
+        "message": "Your receiver has just recovered from receiver failsafe but the arm switch is on",
+        "description": "Description of the BAD_RX_RECOVERY arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipBOXFAILSAFE": {
+        "message": "The 'FAILSAFE' switch was activated",
+        "description": "Description of the BOXFAILSAFE arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipRUNAWAY_TAKEOFF": {
+        "message": "Runway Takeoff Prevention has been triggered",
+        "description": "Description of the RUNAWAY_TAKEOFF arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipTHROTTLE": {
+        "message": "Throttle channel is too high",
+        "description": "Description of the THROTTLE arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipANGLE": {
+        "message": "Craft is not level (enough)",
+        "description": "Description of the ANGLE arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipBOOT_GRACE_TIME": {
+        "message": "Arming too soon after power on",
+        "description": "Description of the BOOT_GRACE_TIME arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipNOPREARM": {
+        "message": "Prearm switch is not activated or prearm has not been toggled after disarm",
+        "description": "Description of the NOPREARM arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipLOAD": {
+        "message": "System load is too high for safe flight",
+        "description": "Description of the LOAD arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipCALIBRATING": {
+        "message": "Sensor calibration is still ongoing",
+        "description": "Description of the CALIBRATING arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipCLI": {
+        "message": "CLI is active",
+        "description": "Description of the CLI arming disable flag"
+    },
+    "initialSetupArmingDisableFlagsTooltipCMS_MENU": {
```

Comment:
Looking at this and `initialSetupArmingDisableFlagsTooltipOSD_MENU` makes me suspect that these are in fact redundant, and that `initialSetupArmingDisableFlagsTooltipOSD_MENU` and moreso `OSD_MENU` as an arming disabled reason can probably be eliminated:

https://github.com/betaflight/betaflight/blob/e020bcca9ef83b2cfad4df3812dabed552ae3bfd/src/main/io/osd.c#L2048-L2052

means that arming is only disabled with `ARMING_DISABLED_OSD_MENU` when `displayIsGrabbed(osdDisplayPort)`. 

https://github.com/betaflight/betaflight/blob/e020bcca9ef83b2cfad4df3812dabed552ae3bfd/src/main/drivers/display.c#L67-L70

and 

https://github.com/betaflight/betaflight/blob/e020bcca9ef83b2cfad4df3812dabed552ae3bfd/src/main/drivers/display.c#L48-L53

mean we need to call `displayGrab()` for this.

And

https://github.com/betaflight/betaflight/blob/e020bcca9ef83b2cfad4df3812dabed552ae3bfd/src/main/cms/cms.c#L687-L708

means that we only ever do that when arming is already disabled.

@etracer65, @jflyper: What's your thoughts on this?

@McGiverGim: The code in this pull request looks good, but I think we should wait with mergeing this until we've sorted this out, and, if possible, eliminated OSD_MENU as an arming disabled flag.